doc/api/http.markdown

```diff
@@ -43,12 +43,6 @@ list like the following:
       'Host', 'mysite.com',
       'accepT', '*/*' ]
 
-## http.METHODS
-
-* {Array}
-
-A list of the HTTP methods that are supported by the parser.
-
```

Comment:
There is a decent amount of code that accesses this. Maybe we should deprecate in docs first? 

There is a full replacement available [[1]](https://www.npmjs.com/package/methods), but if we are still using the list in core, is there a reason to not export it?